x/commitment/keeper/msg_server_deposit_tokens.go

```diff
@@ -56,5 +56,18 @@ func (k msgServer) DepositTokens(goCtx context.Context, msg *types.MsgDepositTok
 	// Update the commitments
 	k.SetCommitments(ctx, commitments)
 
+	// Emit Hook commitment changed
+	k.HookCommitmentChanged(ctx, msg.Creator, sdk.NewCoin(msg.Denom, msg.Amount))
```

Comment:
Can you change the name to `OnCommitmentChange` or `AfterCommitmentChange` - normal hook names look like that.

Reply:
Okay noted.